subseas_flex_fcst/maproom.py

```diff
@@ -39,6 +46,53 @@
 
 APP.layout = layout.app_layout
 
+
+def adm_borders(shapes):
+    with psycopg2.connect(**CONFIG["db"]) as conn:
+        s = sql.Composed(
+            [
+                sql.SQL("with g as ("),
+                sql.SQL(shapes),
+                sql.SQL(
+                    """
+                    )
+                    select
+                        g.label, g.key, g.the_geom
+                    from g
+                    """
+                ),
+            ]
+        )
+        df = pd.read_sql(s, conn)
+
+    df["the_geom"] = df["the_geom"].apply(lambda x: wkb.loads(x.tobytes()))
+    df["the_geom"] = df["the_geom"].apply(
+        lambda x: x if isinstance(x, MultiPolygon) else MultiPolygon([x])
+    )
```

Comment:
This step (69-71) should no longer be necessary, since https://github.com/iridl/python-maprooms/pull/198 . Please verify.